### PR TITLE
#49 Implement Intro page

### PR DESCRIPTION
**Pull request by zelenyhleb**
Implemented customizable xhtml-based eclipse intro for further frontend development.

Signed-off-by: Nikifor Fedorov <zelenyhleb@gmail.com>

### CODE REVIEW

File: bundles/org.openhwgroup.corev.ide.ui.intro/content/styles/bootstrap.min.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*!
+ * Bootstrap v4.3.1 (https://getbootstrap.com/)
+ * Copyright 2011-2019 The Bootstrap Authors
```
Comment (ruspl-afed) [inline bundles/org.openhwgroup.corev.ide.ui.intro/content/styles/bootstrap.min.css:3]: Perhaps we should create a CQ for this, let's remove it from this PR

File: bundles/org.openhwgroup.corev.ide.ui.intro/plugin.xml [hunk 2]
```diff
@@ -13,11 +13,30 @@
...
+		</intro>
+		<introProductBinding
+		 	introId="org.openhwgroup.corev.ide.intro"
+		 	productId="org.openhwgroup.corev.ide.product">
+		</introProductBinding>
+	</extension>
+	<extension
+         point="org.eclipse.ui.intro.config">
+      <config
+            id="org.eclipse.passage.loc.configId"
```
Comment (ruspl-afed) [inline bundles/org.openhwgroup.corev.ide.ui.intro/plugin.xml:29]: id seems to be wrong

**Review by ruspl-afed — CHANGES_REQUESTED** [review]
Please address the review comments

**Comment by zelenyhleb** [comment]
Current state of intro page:
Screenshot

**Review by ruspl-afed — APPROVED** [review]

**Comment by ruspl-afed** [comment]
Thanks @zelenyhleb

**Pull request merged**